docs/xy-plot.md

```diff
@@ -54,6 +54,24 @@ XYPlot is a wrapper for series, hints, axes and other components. Most of these
 * `size` (optional)
 * `style` (optional) - css properties as an object.
 
+Accessors can also be used to retreieve the properties above from the `data` object. For instance, the `getX` and `getY` accessors can be passed to the XYPlot object to access the `x` and `y` properties from `data` for each series. 
+
+```jsx
+<XYPlot
+  width={300}
+  height={300}
+  getX={d => d[0]}
+  getY={d => d[1]>
```

Comment:
I know that this PR isn't implementing accessors, but now I'm curious–do we also support accessors on series components? That seems like a more common use case that people might have (and if we support it we should document it).

```
const data = [
  {fruit: 'apple', bought: 23, sold: 18},
  {fruit: 'banana', bought: 28, sold: 27}
];

<XYPlot
   ...
   getX{d => d.fruit}
/>
   <LineSeries getY={d => d.bought} />
   <LineSeries getY={d => d.sold} />
</XYPlot>
```

@jckr @mcnuttandrew

Reply:
Series level accessors *might* be supported, though if they are the functionality is untested. The only guaranteed support that we offer is on a plot level. I'd support adding accessors on series, seems like a reasonable idea.

Reply:
I don't think they are supported. the accessors seem to be passed down from `XYPlot`.